### DISCUSSION TITLE
List breaking changes on abp update (CLI)

### DISCUSSION THREAD

**Discussion by nebula2** [abpframework/abp#13392] [Ideas]
Just a random thought:

When performing an update of abp through cli, list breaking changes or link to release docs to (hopefully) reduce amount of issues raised.

**Comment by enisn** [comment]
We've published migration guide, that may help:
https://docs.abp.io/en/abp/6.0/Migration-Guides/Abp-6_0

**Reply by nebula2** [reply]
Jeah that is done on every release. And yet there are issues for some reason.

Therefore this idea - to list the breaking changes when performing an update through cli.
